src/mole_C++/operators.h

```diff
@@ -1,3 +1,11 @@
+/**
+ * @file operators.h
+ * 
+ * @brief Sparse operation inline definitions for mimetic members
```

Comment:
What are _mimetic members_?

Reply:
I don't know. Wether to call them constructors or members is still unclear. But, they are called class constructors now.
https://github.com/csrc-sdsu/mole/blob/0295c8fdbeebed3a51260cb1276d185a8270844d/src/mole_C%2B%2B/operators.h#L1-L7

Reply:
What about: Overloads for Mimetic Operators and common linear algebra objects.